homeTask6/src/main/java/org/example/servlets/AddStudentsServlet.java

```diff
@@ -0,0 +1,70 @@
+package org.example.servlets;
+
+import org.example.driver.MySQLDriverManager;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+
+public class AddStudentsServlet extends HttpServlet {
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
+        getServletContext().getRequestDispatcher("/WEB-INF/AddStudent.html").forward(req, resp);
+    }
+
+    @Override
+    protected void doPost(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
+        MySQLDriverManager driverManager = MySQLDriverManager.getInstance();
+        Connection connection = null;
+        PreparedStatement preparedStatement = null;
+
+        try {
+            connection = driverManager.getConnection();
+            connection.setAutoCommit(false);
+        } catch (SQLException e) {
+            throw new RuntimeException(e);
+        }
+
+        try {
+            preparedStatement = connection.prepareStatement("INSERT INTO table2.students values (?,?,?,?,?)");
+            preparedStatement.setString(1, "id");
```

Comment:
Actually, according to the task, we should put here values, which we retrieve from the request